TUTORIAL.md

```diff
@@ -159,6 +159,40 @@ If you encounter 'Connection reset by peer' errors when cloning, run, say, `assi
 (Assigner doesn't have plans for any grading features;
 however, if you are interested in automated grading, check out its sister project, [grader](https://github.com/redkyn/grader).)
 
+### Uploading scores to Canvas
+
+Although Assigner does not support grading, it can be used to upload the scores generated by an automated grader to Canvas.
+Currently, this feature is supported for workflows in which an automated grading script runs as part of a Continuous Integration (CI) job and 
+produces an output file (artifact) containing the score.
+
+There are two methods of uploading these scores:
+1. Use `assigner score all` if you'd like to get the scores for all students, passing the `--upload` flag if you'd like to upload them to Canvas.
+1. Use `assigner score checkout` if you'd like to upload scores one at a time, using an interactive search function to select a student.  
+
+The latter option may be useful in circumstances where an instructor wants to verify attendance as part of a programming lab or test.
+
+An example grading workflow for a conventional assignment might look like the following:
+1. `assigner score all assignment-name` while the assignment is open, to monitor student progress.
+1. `assigner lock assignment-name` once the due date is reached
+1. `assigner score all --upload assignment-name -f path/to/grader_file1 path/to/grader_file2` to upload scores and check whether grading-related files were modified 
+by the student
```

Comment:
This is a good example! It would be perhaps more helpful to explain what `path/to/grader_file1` &c. are.